abel/basex.py

```diff
@@ -26,8 +26,12 @@
 # V. Dribinski, A. Ossadtchi, V. A. Mandelshtam, and H. Reisler,
 # Review of Scientific Instruments 73, 2634 (2002).
 #
-# 
-# version 0.62 - 2016-03-07 
+#
+# Version 0.63 - 2018-09-08
```

Comment:
These "version" label are no longer needed. You can remove them and just have the date and the change.